src/boot/efi/vmm.c

```diff
@@ -289,3 +295,21 @@ bool in_hypervisor(void) {
         cache = cpuid_in_hypervisor() || smbios_in_hypervisor();
         return cache;
 }
+
+const char* smbios_find_oem_string(const char *name) {
+        assert(name);
+
+        SmbiosTableType11 *type11 = (SmbiosTableType11 *) get_smbios_table(11);
```

Comment:
hmm, there should be a lot of `const` be springled around here. smbios stuff is read-only after all (at least from our perspective)